setup.py

```diff
@@ -19,6 +19,7 @@
     "sentencepiece",
     "scipy",
     "neural_compressor",
+    "openvino",
```

Comment:
there can be potential issues between INC and OpenVINO dependencies. I mean they can be incompatible. One way to workaround this problem is to introduce extras, so that the user will install the package like, `pip install optimum[intel, openvino]`.